Makefile

```diff
@@ -937,12 +945,22 @@ set-manifest-image:
 clean: ## Remove all generated files
 	$(MAKE) clean-bin
 	$(MAKE) clean-book
+	$(MAKE) clean-tilt
+	$(MAKE) clean-charts
 
 .PHONY: clean-bin
 clean-bin: ## Remove all generated binaries
 	rm -rf $(BIN_DIR)
 	rm -rf $(TOOLS_BIN_DIR)
 
+.PHONY: clean-tilt
+clean-tilt: ## Remove all files in .tiltbuild
+	rm -rf ./tiltbuild
+
+.PHONY: clean-charts
+clean-charts: ## Remove all local copies of Helm charts in ./hack/observability
+	(for path in "./hack/observability/*"; do rm -rf $$path/charts ; done)
```

Comment:
The clean targets below follow something like this with IFS, is that something we need here or is a simple loop like this enough?
```
(IFS=','; for i in $(SRC_DIRS); do find $$i -type f -name '*.yaml' -exec rm -f {} \;; done)
```

Reply:
The current change seems to be working as expected and it should be fine.